src/diffusers/models/attention.py

```diff
@@ -280,6 +281,7 @@ def __init__(
         super().__init__()
         inner_dim = int(dim * mult)
         dim_out = dim_out if dim_out is not None else dim
+        linear_cls = LoRACompatibleLinear if not USE_PEFT_BACKEND else nn.Linear
```

Comment:
This line is repeated multiple times. I wonder if it would make sense to define a constant `Linear = LoRACompatibleLinear if not USE_PEFT_BACKEND else nn.Linear` and import + use that everywhere? It would reduce code duplication but it's less transparent, so I'm not sure.

Same argument applies to `Conv2d`.

Reply:
Hmm not sure if we can really do that because i think that not all linear layers use `LoRACompatibleLinear`